src/components/Categories.js

```diff
@@ -0,0 +1,7 @@
+const Categories = () => {
+  <div className="wrapper">
+    <h1>Under construction</h1>
+  </div>;
+};
```

Comment:
- [ ] good job moving this component to the right directory as the previous CR mentioned however the application is still not working, because it is expecting a return statement when you put the code between curly brackets, what you can do to fix this issue is putting the code between normal brackets, this will look something like below:
```
const Categories = () => (
  <div className="wrapper">
    <h1>Under construction</h1>
  </div>
);
```